metadata.rb

```diff
@@ -8,3 +8,5 @@
 
 supports         'redhat'
 supports         'centos'
+
+depends          'python', '= 1.4.6'
```

Comment:
Is there a need to add this hard dependency? Or could we do with something like `'python', '>1.4.0'` (or whatever version is the minimal needed version)?

And... I noticed this line in the cookbook readme: `THIS COOKBOOK IS DEPRECATED` adn it says the better cookbook to use is https://github.com/poise/poise-python so I suggest we do that right?